flight_phase_files/HALO/EUREC4A_HALO_Flight-Segments_20200215.yaml

```
@@ -279,7 +281,7 @@ segments:
 - kinds:
   - circling
   name: circling 3
-  irregularities: [] 
+  irregularities: []
```

I'd rather not do a white space change here, as this makes it more complicated to find out how the file has been made.
```suggestion
  irregularities: [] 
```

---

> * You have added two irregularities (i.e. two list items) to each of the circle, which in principle is fine. However I don't see which is the distinctive separation between those items. In particular, at `HALO-0119_c1` center and radius are separated while in `HALO-0215_c3` they are both within one irregularity

That was probably due to me not thinking about the structure too much. So I'd be fine with including everything into one irregularity.

> * Do we need an irregularity tag? Something comparable to `TTFS` like `NONSTANDARD` or `NONSTD` or the like? This would simplify to filter out all non-standard circles in a script.

That is a good idea. It is however only the case for two circles. So I'm not sure if this might be too much for just two cases? But maybe it doesn't hurt either. Sorry, no clear vote from me here :-) I would leave the decision up to the maintainers what seems more reasonable.



> I'd rather not do a white space change here, as this makes it more complicated to find out how the file has been made.

That is ok. This change was probably done by the editor automatically (I don't remember changing anything there ;-) ) Sorry for the confusion.

Oh and one more thing I didn't mention in the original text above: The circle centers are calculated as
`[mean([min(lat), max(lat)]), mean([min(lon), max(lon)])]`
from BAHAMAS position data. So this highly depends on the position data that make up a circle. But in the two cases reported it seems to be reasonably accurate to do so.

---

Ok, then my suggestion is as follows:
* put everything in one list item per circle
* prepend `NONSTD` in front of each line
* add an item about `NONSTD` to the end of `README.md`, i.e.: `NONSTD`: segment is not at its usual location or size (only added if there is a usual location or size for this segment `kind` on that `platform`)
* re-introduce the whitespace

I am not sure about if we should place the computation of the circle center somewhere. I tend not to put it anywhere and assume that people would find this place if they really care.